ui/app/components/ens-input.js

```diff
@@ -85,14 +85,6 @@ EnsInput.prototype.lookupEnsName = function () {
   const recipient = document.querySelector('input[name="address"]').value
   const { ensResolution } = this.state
 
-  if (!this.ens) {
-    return this.setState({
-      loadingEns: false,
-      ensFailure: true,
-      hoverText: 'ENS is not supported on your current network.',
-    })
-  }
```

Comment:
What does removing this do, when trying an `.eth` address on a non-eth network? Just show "no address for this name"?